Scripts/Multis/Houses/Gumps/HouseGump.cs

```diff
@@ -1576,7 +1576,6 @@ namespace Server.Prompts
 {
     public class RenamePrompt : Prompt
     {
-        public override int MessageCliloc => 501302;
```

Comment:
Have you tested on EC? That function affects EC.

Reply:
No I did not.

Can you test on EC quickly? I do not have it installed at the moment.

If you search for the cliloc number 501302 you will see it is applied above.

Reply:
before 
![housess](https://user-images.githubusercontent.com/17765272/88551799-f624cf00-d02b-11ea-9956-342db4ee789f.PNG)

after
![housess2](https://user-images.githubusercontent.com/17765272/88551805-f755fc00-d02b-11ea-8204-11469651d9c3.PNG)

you just need to delete it.

from.SendLocalizedMessage(501302); // What dost thou wish the sign to say?